tests/test_api.py

```diff
@@ -104,7 +104,7 @@ def test_non_existant_api(self):
         assert response.json['data'] == {}
         assert response.json['code'] == 404
 
-    def test_aaaget_backup(self):
+    def test_get_backup(self):
```

Comment:
was this named `test_aaget` for a reason? I suspect it was (maybe to get the test to run first?)